internal/clusterinfo/data.go

```diff
@@ -119,24 +119,57 @@ func (c *ClusterInfo) GetLookupdTopics(lookupdHTTPAddrs []string) ([]string, err
 
 // GetLookupdTopicChannels returns a []string containing a union of all the channels
 // from all the given lookupd for the given topic
-func (c *ClusterInfo) GetLookupdTopicChannels(topic string, lookupdHTTPAddrs []string) ([]string, error) {
-	var channels []string
+func (c *ClusterInfo) GetLookupdTopicChannels(topic string, lookupdHTTPAddrs []string) (*TopicMeta, *TopicChannelsMeta, error) {
```

Comment:
We can continue to simplify the return values here, since it's an internal API. How about:

```go
type ChannelState struct {
    Name string
    Paused bool
}

func (c *ClusterInfo) GetLookupdTopicChannels(topic string, lookupdHTTPAddrs []string) ([]ChannelState, error) {
```

I don't think it makes sense to force a return value that includes topic metadata.